go-controller/pkg/util/ovs.go

```diff
@@ -182,7 +183,7 @@ func runOVNretry(cmdPath string, args ...string) (*bytes.Buffer, *bytes.Buffer,
 			time.Sleep(2 * time.Second)
 		} else {
 			// Some other problem for caller to handle
-			return stdout, stderr, err
+			return stdout, stderr, errors.New(err.Error() + "Actual OVN cmd: " + cmdPath + " " + strings.Join(args, " "))
```

Comment:
Can you do this instead?
`return stdout, stderr, fmt.Errorf("OVN command '%s %s' failed: %s",  cmdPath, strings.Join(args, " "), err)`

Reply:
Thank you Girish for the go tip :)
pushed.